src/libs/axios/index.tsx

```diff
@@ -30,6 +32,7 @@ axiosClient.interceptors.response.use(
     if (err.response.status === 403) {
       window.location.href = '/sign-in';
       localStorage.removeItem('TOKEN');
+      window.location.href = Path.PRODUCTION_MONITORING;
```

Comment:
Why do we set location.href twice?

Reply:
I got mistake when resolve conflict from master,

Reply:
Dai bu cac cho